MODULE.bazel

```diff
@@ -13,6 +13,18 @@ bazel_dep(name = "lz4", version = "1.9.4")
 bazel_dep(name = "platforms", version = "0.0.10")
 bazel_dep(name = "rules_cc", version = "0.0.9")
 bazel_dep(name = "rules_foreign_cc", version = "0.11.1")
+bazel_dep(name = "rules_oci", version = "2.0.0")
+
+oci = use_extension("@rules_oci//oci:extensions.bzl", "oci")
+oci.pull(
+    name = "rules_ros_base_image",
+    image = "python:3.10-bookworm",
+    digest = "sha256:428430918ab72109921ef55d5f8bdd3a02a90734fd79ee920146cf385a13c110",
+    platforms = ["linux/amd64"],
+)
+use_repo(oci, "rules_ros_base_image", "rules_ros_base_image_linux_amd64")
+
+bazel_dep(name = "rules_pkg", version = "1.0.1")
```

Comment:
With the refactorings made in https://github.com/mvukov/rules_ros/pull/43, we might want to move this into `examples/MODULE.bazel`

Reply:
done